### DISCUSSION TITLE
Library Source Material - CDE 14808227 (retired)

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#114] [Metadata Field Ratification]
caDSR Link: [14808227](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1)

Definition: "The cellular source of the double stranded DNA fragments analyzed by high-throughput sequencing."

**Comment by GeoffLyle** [comment]
Note: Splitting this from previous discussion #46, as that discussion was tracking both **library source material** and **library source molecule**.  As **library source material** has been approved by the data harmonization team for implementation, a separate discussion to track its implementation status is warranted.

**Comment by Ssandor13** [comment]
From October 04, 2024 meeting:

Although this CDE was approved by the harmonization subgroup, we are re-opening this CDE to discuss what the next steps should be to map heme samples. 

Per the discussion in [#46](https://github.com/CBIIT/ccdi-federation-api/discussions/46) it was agreed to map to Bulk Tissue, however, the Federation agreed to not map heme samples in the meantime until 1) a new PV is agreed upon and accepted or 2) we find that one of the PV in the current caDSR CDE is adequate enough to map to. 

At this time, mapping to Bulk Tissue seems as too much of a stretch to map to and thus this use case will be re-interviewed.

**Reply by GeoffLyle** [reply]
For blood samples, do you have a preferred term or a term that is used internally?
If you have a preferred term perhaps @jknable or @dunnpa could find a CDE that fits better.

Treehouse would use "Bulk Tissue" for all of our RNA-Seq samples.  While I don't believe any current members are sharing Single Cell data, I'm thinking the primary benefit of this CDE will be to differentiate between Bulk and Single Cell data.  @jknable @dunnpa Are there other CDEs the CCDI is using to differentiate between Bulk and Single Cell data?

**Reply by dunnpa** [reply]
CCDI uses the ICD-O-3.2 Topology code and term C42.0 : Blood in its anatomic_site property 14156279.
To distinguish bulk tissue from single cell, CCDI uses library_source_material 14808227 as edited by caDSR with Federation input and the following PVs:

Bulk Cells
Bulk Nuclei
Bulk Tissue
Single-cells
Single-nuclei
Not Reported
Other

**Reply by Ssandor13** [reply]
Based on the provided permissible values for this [CDE](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1), none seem to directly correspond to blood as a cellular material source, as we discussed. Blood typically contains multiple cell types like red blood cells, white blood cells (buffy coat), and platelets, and can be processed into different preparations like bulk blood cells or plasma.

A potential new permissible value could be:
**VM Long Name**: Bulk Blood Cells
**Definition**: A biospecimen derived from whole blood consisting of multiple cell types, intended to be analyzed as a pool.
This definition would capture the essence of blood as a source while aligning it with the format of other permissible values like "Bulk Cells" and "Bulk Tissue. @jknable @GeoffLyle

**Reply by dunnpa** [reply]
Uberon has `'blood` 
http://purl.obolibrary.org/obo/UBERON_0000178
which can be interpreted as whole blood vs the many blood related derivatives. 

I suggest not using bulk blood as that conflates the bulk vs single cell classification with the anatomic site or specimen type.

**Reply by Ssandor13** [reply]
From October 18th, 2024 meeting:

The team further discussed this and agreed with the consensus to map `Blood` to `Bulk Tissue` for this CDE.

**Reply by GeoffLyle** [reply]
@dunnpa
I'm seeing that "Other" is not a permissible value on the caDSR website (
https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1).

When we implement this CDE, we will not be adding "Other" as it is not a listed permissible value.  Is "Other" going to be added to this CDE?

**Reply by dunnpa** [reply]
I think this PV list is pretty good as is.
Janice and Maureen, what are your thoughts on adding "Other" as a PV?

**Comment by dunnpa** [comment]
From: "Ryan, Maureen (NIH/NCI) [C]" <maureen.ryan2@nih.gov>

Hi Geoff,


The term “Blood Bulk Cells” is actually combining the specimen, blood, with the source material, bulk cells.  There are two separate CDEs for these two properties. 

•	For the specimen, blood, please use CDE 12445832 Specimen Material Category.
•	For the library source material, bulk cells, please use CDE 14808227 Library Source Material.

**Comment by GeoffLyle** [comment]
At November 1, 2024, meeting it was decided to use the new CDE #119. CDE [14808227](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=14808227%20and%20ver_nr=1) will not be implemented in the CCDI Federation open API.
